pkg/azurecluster/validate_update.go

```diff
@@ -36,6 +36,13 @@ func (h *WebhookHandler) OnUpdateValidate(ctx context.Context, oldObject interfa
 	err = errors.IgnoreCAPIErrorForField("spec.SubscriptionID", err)
 	err = errors.IgnoreCAPIErrorForField("spec.ControlPlaneEndpoint.Host", err)
 	err = errors.IgnoreCAPIErrorForField("spec.ControlPlaneEndpoint.Port", err)
+	isCapi, lerr := generic.IsCAPIRelease(azureClusterOldCR)
```

Comment:
azure-admission-controller is ignoring all CRs from CAPI releases, so this check should not be required

Reply:
e.g. https://github.com/giantswarm/azure-admission-controller/blob/master/pkg/validator/http_handler_factory.go#L71-L83

Reply:
ok removed